pkg/connector/user.go

```diff
@@ -325,3 +219,161 @@ func extractFromInterface[T any](data interface{}) ([]T, error) {
 
 	return result, nil
 }
+
+func (o *userResourceType) Get(ctx context.Context, resourceId *v2.ResourceId, parentResourceId *v2.ResourceId) (*v2.Resource, annotations.Annotations, error) {
+	l := ctxzap.Extract(ctx)
+
+	r := o.userService.Users.Get(resourceId.Resource).Projection("full")
+
+	user, err := r.Context(ctx).Do()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	if o.domain != "" {
```

Comment:
This looks like net new code and doesn't seem to be just partial sync related? Are we sneaking in something else along side of the partial sync sdk update? This is to only get user that match a configured domain or customer id I think.

Reply:
Nevermind, I see we had this before:

```
if o.domain != "" {
		r = r.Domain(o.domain)
	} else {
		r = r.Customer(o.customerId)
	}
	```